hack/run-e2e.sh

```diff
@@ -0,0 +1,14 @@
+set -o errexit
+set -o nounset
+set -o pipefail
+
+DESCHEDULER_OPERATOR_ROOT=$(dirname "${BASH_SOURCE}")/..
+
+# Get operator-sdk binary.
+# TODO: Make this to download the same version we have in godeps
+wget -O /tmp/operator-sdk https://github.com/operator-framework/operator-sdk/releases/download/v0.2.0/operator-sdk-v0.2.0-x86_64-linux-gnu
```

Comment:
This is more ci-specific, if someone wants to run locally, there needs to be a way for it. One alternative is to add kubeconfig flag, if nothing provided run in CI mode where we assume kubeconfig is present at a particular path.